src/codeu/chat/client/core/Controller.java

```diff
@@ -39,6 +39,80 @@ public Controller(ConnectionSource source) {
     this.source = source;
   }
 
+  // TODO: add return options, no user found, no access,
```

Comment:
Implement this TODO or remove it if it does not necessary to complete this feature.